torch/jit/__init__.py

```diff
@@ -772,6 +782,11 @@ def check_unique(param):
 
         self._freeze()
 
+    def forward(self, *args, **kwargs):
+        if self._has_method('forward'):
+            return self._get_method('forward')(*args, **kwargs)
+        raise RuntimeError('Trace submodules cannot be called.')
```

Comment:
Would be faster to use `_get_method` in `try ... except` and raise a readable error if this returns a KeyError or sth like this